collectors/statsd.plugin/statsd.c

```diff
@@ -818,7 +818,7 @@ static void statsd_process_metric(const char *name, const char *value, const cha
             statsd_parse_field_trim(tagkey, tagkey_end);
             statsd_parse_field_trim(tagvalue, tagvalue_end);
 
-            if(tagkey && tagkey && tagvalue && *tagvalue) {
+            if(tagvalue && *tagkey && *tagvalue) {
```

Comment:
I think I just forgot to add the asterisk before the second `tagvalue`. We should check for both not NULL and not empty for both strings.